docs/antrea-network-policy.md

```diff
@@ -671,6 +718,7 @@ can be set for a ClusterGroup, i.e. a single ClusterGroup can either group workl
 represent IP CIDRs or select other ClusterGroups. A parent ClusterGroup can select different
 types of ClusterGroups (Pod/Service/CIDRs), but as mentioned above, it cannot select a
 ClusterGroup that has childGroups itself.
+- A ClusterGroup cannot be set with `namespaces` field.
```

Comment:
in which context? In the context of an ACNP peer? If yes, this seems like a strange place to mention it

Reply:
removed